src/libraries/iCubDev/include/iCub/rawValuesPublisher.thrift

```diff
@@ -0,0 +1,38 @@
+# Copyright: (C) 2024 iCub Facility - Istituto Italiano di Tecnologia
+# Authors: Losi Jacopo, valentina Gaggero
+# CopyPolicy: Released under the terms of the GNU GPL v2.0.
+#
+# rawValuePublisher.thrift
+
```

Comment:
```suggestion
namespace iCub
```

Adding a namespace here should ensure that all the classes are generated in the correct iCub namespace.

Reply:
(I updated the comment for more details).